wordpress-mobile/WordPress-iOS
# Issues/6332 flag and preserve inuse posts

Fixes #6332 

To test:
Repeat the steps outlined in #6297.
Confirm that the post is not deleted (watch for the NSLog or set a breakpoint), that instead its topic is set to nil, and there is no crash.

Needs review: @koke would you mind since you are familiar with the issue?

Note: State restoration appears recently broken in develop on the iPhone (Swift 3 merge maybe?) so try testing via the iPad.
